release-drafter & co also in master branch

peppelinux opened the pull request:
This should fix the current CI

feat: release-drafter also in master branch

rohe commented:
Do you understand why all tests fail. And they fail with the message that the directory tests doesn't exist.

peppelinux commented:
> Do you understand why all tests fail. And they fail with the message that the directory tests doesn't exist.

because in master we don't have tests folder.
I just removed the gh actions from this branch and removed the main branch from release-drafter configuration.

It just have to be merged in master, then release-drafter won't fail anymore ... My theory
